extensions/packageManager/src/node/index.ts

```diff
@@ -352,7 +351,7 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
 
       let runtimePath = currentProject.settings?.runtime?.path;
       if (runtimePath && !path.isAbsolute(runtimePath)) {
-        runtimePath = path.resolve(currentProject.dir, runtimePath);
+        runtimePath = path.resolve(currentProject.dir, 'settings', runtimePath);
       }
```

Comment:
I'm a little worried about the amount of duplicated code. Can we add a method to the project that centralizes this?

Reply:
good idea.

Reply:
@a-b-r-o-w-n I addressed this.  Want to give it another look?